### ISSUE TITLE
Dice mean of the validation set is decreased, and dose the Brain Dice mean when the training step increases

### ISSUE THREAD

**Issue by pypi20200320** [cwmok/C2FViT#9]
Thanks for your excellent work. But I encounted some problem.
First I trained a C2FViT model use my own dataset based on the script ,Train_C2FViT_pairwise.py. But there is no downward trend of the NCC-loss and the Dice mean of the validation set is decreased. I think maybe NCC is not suit for my dataset(which is CBCT2CT)，so I use MSE-loss replace of NCC-loss, then the loss is decreasing and dice mean of validation set is increasing, from 0.74 to 0.86.

It's so confused.

And then I trained  a C2FViT model in an unsupervised manner for pairwise registration with the OASIS dataset use the  original script, Train_C2FViT_pairwise.py. But the Dice mean of the validation set is gradually decreased, and dose the Brain Dice mean when the training step increases. I can't figure out what's the problem because I totally have not changed your code.

Can you figure out why is this situation happening ?
